.github/workflows/gateway-conformance.yml

```
@@ -0,0 +1,146 @@
+name: Gateway Conformance Dashboard
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  # Enable this when we're ready to generate the dashboard
+  # schedule:
+  #   - cron: "0 */6 * * *" # every six hours
+
+concurrency:
+  group: "conformance"
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  configure:
+    runs-on: ubuntu-latest
+    outputs:
+      gateways: ${{ steps.set-matrix.outputs.gateways }}
+    steps:
+     - name: Checkout repository
+       uses: actions/checkout@v3
+     - name: install yq
+       env:
+          VERSION: v4.35.1
+          BINARY: yq_linux_amd64
+       run: |
+        wget https://github.com/mikefarah/yq/releases/download/${VERSION}/${BINARY} -O /usr/local/bin/yq
+        chmod +x /usr/local/bin/yq
+     - name: Set matrix data
+       id: set-matrix
+       run: |
+        echo "gateways=$(yq -o=json -I=0 '.' gateways.yml)" >> $GITHUB_OUTPUT
```

I'm not sure how this yq works. Does it output compact list i.e. a single line? If not, setting the output val has to be handled a bit differently.

Apart from that, I quite like using xargs and tee through pipes because:
- the step fails when yq fails
- you get the value on stdout which is helpful when debugging

```suggestion
        yq -o=json -I=0 '.' gateways.yml | xargs -I {} echo "gateway={}" | tee -a $GITHUB_OUTPUT
```

---

I made that line work for our case, but it's trickier than expected.
Might be good to know:

```sh
xargs -0 -S2000000 -I {} echo "gateways={}"
```

- `-0` to skip interpretation without it, xargs will drop the quotes from our json input
- `-S200000` to allow larger outputs (255 by default), without it, xargs will ignore the transformation and render `{}` as is. No exit code either. The number I picked is `current file size x 1000` so roughly 81 -> 80k gateways.

Which gives in the end
```sh
        jq --raw-input --slurp --compact-output 'split("\n")[:-1]' gateways.txt | \ # related to using txt instead of yaml
          tr -d '\n' | \ # required when using `-0` below
          xargs -0 -S2000000 -I {} echo "gateways={}" | \ # the line
          tee -a "$GITHUB_OUTPUT" # finally!
````